_posts/zman7895/2017-05-22-zman7895-turtle-hack.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: zman7895
+title: "zman7895's turtle hack"
+---
+Below is my embedded Turtle Hack program.
+<iframe src="https://trinket.io/embed/python/ccad0f5a12" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I knew from the very beginning I wanted to implement a competitive game for my turtle hack, mainly because I am a competitive person. I decided I was going to do a trivia game between two players. This was all in my mind playing out without being in front of the project. Later in the evening I was watching the NBA Playoffs, and I liked the idea of having the game be a best of 7 like the series are in the NBA. I realized in order to do this, I would need 7 questions, and someone would have to win each round, to make it a best of 7. I didn't know how I was going to do this, but I had my overall idea to sleep on. It was the next day that I sat down in front of my project. I first created the graphic of a black screen with a white finish line. Then I created two turtles to appear as opposing players. I utilized the input method we learned in the exercise to ask players the trivia questions. All of this came pretty easily just off of what we had done in class. Then things got interesting.
```

Comment:
This description of your process is excellent, as is the process itself.  Thinking about code without a computer is super powerful!

With your coding process, you also did a great job of segmenting off the setup, which you knew how to do, from the logic and question asking part.  This let you focus on the crux of the game logic specifically.